packages/clerk-js/src/ui/common/Billing/ManageBillingPlanScreen.tsx

```diff
@@ -98,15 +98,15 @@ export type OrganizationPlanCardProps = Pick<BillingPlanResource, 'name' | 'feat
 
 export const OrganizationPlanCard = (params: OrganizationPlanCardProps) => {
```

Comment:
Not related to this PR but I think we should rename this to  `OrganizationPlanCard` => `PlanCard`